### PR TITLE
LPS-177334 | Test fix | Master

### PR DESCRIPTION
**Pull request by GABRIELCMSANTOS**
[LPS-177334](https://issues.liferay.com/browse/LPS-177334)

### CODE REVIEW

**Comment by GABRIELCMSANTOS** [comment]
ci:test:sf

**Comment by Luizcarlosqueiroz** [comment]
Hey @evanilson-santana ! Would you mind reviewing this PR since you made some suggestions in the ticket, so you got more context about the issue?

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path [hunk 1]
```diff
@@ -20,6 +20,11 @@
 		<td>//button[contains(@class,'btn btn-sm btn-secondary')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ASSERT_ACCOUNT_NAME</td>
+		<td>//div[(@class='dnd-td')]//span[contains(text(),'${key_tabName}')]</td>
+		<td></td>
+	</tr>
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path:27]: Can we use an existing path here?

**Review by timpak — COMMENTED** [review]
@evanilson-santana I don't have context for this fix. Do we still need this? If so, can you take a look to see if we can forward this?

@GABRIELCMSANTOS I left some initial feedback for you. The main problem I see is that the commit message for the fix isn't descriptive so I don't know what we're fixing here. That's not super important but it's something to keep in mind for the future. The commit message should be descriptive enough for anyone to understand what you're trying to accomplish with your changes.

**Comment by evanilson-santana** [comment]
hey @timpak It's duplicate with this [PR](https://github.com/liferay-bpm-qa/liferay-portal/pull/680).

I've talked with @mateussandes23 about this fix and I passed to him the suggestions to fix it. I'll close this PR

**Pull request closed without merging**